src/components/v2/V2Project/index.tsx

```diff
@@ -54,9 +54,10 @@ export default function V2Project() {
   )
 
   const distributionLimitCurrencySymbol =
-    currencyMetadata[distributionLimitCurrency?.toNumber() ?? 1].symbol
+    currencyMetadata[distributionLimitCurrency?.toNumber() ?? 1]?.symbol
```

Comment:
**issue** lets use `V2CurrencySymbol` function here, and also I don't think we should fall back to ETH - we should probably just let it fail.

Reply:
Yeah ok